thoth/prescriptions_refresh/handlers/quay_security.py

```diff
@@ -192,7 +192,11 @@ def _quay_get_security_info(image_name: str, container_id: str) -> List[Dict[str
         headers={"Authorization": f"Bearer {_QUAY_TOKEN}"},
         params={"vulnerabilities": True},
     )
-    response.raise_for_status()
+    if response.status_code != 200:
+        _LOGGER.exception(
+            f"Failed to obtain security information for image {image_name!r} with container id {container_id!r}"
```

Comment:
Maybe make it more clear it was an HTTP error / which one?

```suggestion
            f"Failed to obtain security information from {_QUAY_URL} for image {image_name!r} with container id {container_id!r}: HTTP {response.status_code}"
```

Reply:
Nice catch. I've adjusted the logged error slightly also on another place. PTAL.